src/fixed-footer-layout.tsx

```diff
@@ -34,6 +34,13 @@ const useStyles = createUseStyles((theme) => ({
         width: '100%',
         backgroundColor: theme.colors.background,
         transition: 'background 0.2s linear, box-shadow 0.2s linear',
+        [theme.mq.tabletOrSmaller]: {
+            position: ({hasContentEnoughVSpace, isContentWithScroll}) =>
+                hasContentEnoughVSpace || !isContentWithScroll ? 'fixed' : 'initial',
```

Comment:
the mediaquery placed here works as expected, whereas if placed as a wrapper of the className, it does not work properly 🤷🏼. since it will be moved to vanilla extract, I wouldn't spend much more time on this